docs/setup/native.md

```diff
@@ -97,7 +97,7 @@ the `vets-api` directory, run `./bin/setup-osx && source ~/.bash_profile && cd
        - `touch ./config/certs/vetsgov-localhost.crt`
        - `touch ./config/certs/vetsgov-localhost.key`
 1. Create dev database: `bundle exec rake db:setup`
-1. Go to the file `config/settings/development.yml` in your local vets-api. Switch the commented out lines pertaining to the cache_dir: uncomment out line 14 (what you use for running the app via Rails), and comment out line 15 (what you use for running the app via Docker).
```

Comment:
Seems like since `config/settings/development.yml` is under source control and this is suggesting changing a source controlled file that maybe this setting should be overridden in `config/settings.local.yml`?

Reply:
This line of doc does not read well 😬 .

I agree with your opinion that this should be overridden in `settings.local.yml`. I double-checked the doc and confirmed that settings in `settings.local.yml` will override `settings/development.yml`.

[Source](https://github.com/railsconfig/config#accessing-the-settings-object)

I took a poll a while back to see how many people were using native V.S. docker for local `vets-api` testing, and it was overwhelmingly native.

Reply:
+1 for changing this text to recommend overriding settings in `settings.local.yml`.